app/actions/views/channel.js

```diff
@@ -333,6 +353,12 @@ export function refreshChannel(channelId) {
     };
 }
 
+export function refreshChannelWithRetry(channelId) {
+    return (dispatch, getState) => {
+        getPostsWithRetry(channelId)(dispatch, getState);
```

Comment:
Since we're using the retry methods we watch the actual request status and the retry attempts for refreshing. The retry spinner is based on those now.